### PR TITLE
Add option to hide content behind a non-transparent loading overlay

### LINKED ISSUE #2002: [Enhancement] Make it possible to hide content in the loading overlay component while loading
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
Currently, using the loading overlay will always show the page content behind it. Often we do not wish to do this as the content might not even make sense without the data that is being loaded.

## Describe the solution you'd like
The full page option could be enabled with a input parameter like so
`<kirby-loading-overlay [isLoading]="isLoading" [hideContent]="true">`

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by mark-drastrup**
## Which issue does this PR close?

This PR closes #2002

## What is the new behavior?
It's now possible to hide content behind a non-transparent loading overlay

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.html [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+</button>
+<button
+  kirby-button
+  attentionLevel="2"
+  size="lg"
+  expand="block"
+  (click)="showFullscreenLoadingOverlay(false)"
+>
+  Show full page loading overlay without backdrop
+</button>
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.html]: Could we add this to its own example component instead, and maybe move the buttons above the code viewer so the structure is the same for the two examples?

**Review by RasmusKjeldgaard — DISMISSED** [review]

File: apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.html (hunk 1 above)
Comment (mark-drastrup) [reply apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.html]: Yeah great idea! :D

File: apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.ts [hunk 2]
```diff
@@ -1,8 +1,6 @@
 import { Component } from '@angular/core';
-import { ApiDescriptionProperty } from '~/app/shared/api-description/api-description-properties/api-description-properties.component';
-
 import exampleHtml from '../../examples/loading-overlay-example/loading-overlay-example.component.html?raw';
-
+import { ApiDescriptionProperty } from '../../shared/api-description/api-description-properties/api-description-properties.component';
 @Component({
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.ts:5]: So minor, but there is a missing ~~space~~ newline between the component decorator and the import. Should prettier not take care of that?

**Review by RasmusKjeldgaard — DISMISSED** [review]

File: apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.ts (hunk 2 above)
Comment (mark-drastrup) [reply apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.ts:5]: I can't find any Prettier config options to take care of that 🤔

**Review by RasmusKjeldgaard — APPROVED** [review]
I love it, very nice!

**Pull request merged**